newIDE/app/src/stories/componentStories/QuickCustomization/QuickPublish.stories.js

```diff
@@ -119,6 +124,39 @@ export const AuthenticatedWithTooManyCloudProjects = () => {
     </Template>
   );
 };
+
+export const AuthenticatedWithCloudProjectsMaximumReachedButSavedAlready = () => {
+  return (
+    <Template>
+      <AuthenticatedUserContext.Provider
+        value={{
+          ...fakeAuthenticatedUserWithNoSubscriptionAndCredits,
+          // We have more projects than the maximum allowed, but the project is already saved
+          // so there are no problems.
+          cloudProjects: tenCloudProjects,
+        }}
+      >
+        <QuickPublish
+          project={testProject.project}
+          gameAndBuildsManager={fakeEmptyGameAndBuildsManager}
+          isSavingProject={false}
+          isRequiredToSaveAsNewCloudProject={() =>
+            // Indicates that the project is already saved, there will be
```

Comment:
I feel like this comment should be next to the props type in QuickPublish. I was not sure what it meant before reading this comment

Reply:
Will add it in the type